src/fetch/period_slippage.py

```diff
@@ -134,6 +105,7 @@ def get_period_slippage(
     Executes & Fetches results of slippage query per solver for specified accounting period.
     Returns a class representation of the results as two lists (positive & negative).
     """
+    update_token_list(dune, fetch_trusted_tokens())
```

Comment:
What happens if this is not successful? Will the script stop under all circumstances? E.g. if dune refusing creating a table for some reaon, how would the user be warned?

Reply:
Yes, because fetch is the combination of initiate, execute, await and get results. The method would raise an error if "errors" were contained in the response data from "get_results". Anyway, I have included an assertion in the update method that checks the `token_list` input length is the same as the results fetched from dune.